app/views/alchemy/essences/_essence_file_editor.html.erb

```diff
@@ -46,10 +46,8 @@
         class: 'edit_file',
         title: Alchemy.t(:edit_file_properties) %>
     </div>
-    <% if content.ingredient %>
-      <%= hidden_field_tag content.form_field_name(:attachment_id),
-        content.ingredient.id %>
-    <% end %>
+    <%= hidden_field_tag content.form_field_name(:attachment_id),
+      content.ingredient && content.ingredient.id %>
```

Comment:
The `value` of the form is empty, yes, but the key in the params is still submitted. This was actually the cause of #1096 The params where empty, that's why the essence validations wasn't triggered at all. https://github.com/AlchemyCMS/alchemy_cms/blob/master/app/models/alchemy/element/element_contents.rb#L47 skips the content if no params for it are present.